### DISCUSSION TITLE
Can I get badge

### DISCUSSION THREAD

**Discussion by learningssik** [Schweinepriester/github-profile-achievements#60] [Q&A] [answered]
Can I get badge?

**Comment by nel0x** [comment]
Yeah, let's try something but try selecting next answer or so, not the mine!
(test for https://github.com/Schweinepriester/github-profile-achievements/discussions/45#discussioncomment-3022366)

**Reply by lemonslapper** [reply]
Interesting... @nel0x try to answer now?

**Comment by nel0x** [comment]
Sounds good

**Comment by lemonslapper** [comment]
select @nel0x's next answer as solution

**Comment by nel0x** [comment, marked as answer]
"Lorem ipsum dolor"
https://www.similarweb.com/top-websites/

Please select this answer as the solution @learningssik

**Reply by learningssik** [reply]
Oh…and then can you make discussion and select my answer??

**Reply by melvincwng** [reply]
@learningssik Hi Wonsik, if you want to get the 'Galaxy Brain' badge, please follow the instructions stated in #58.

Hope this is helpful.

Thanks, and have a great day ahead.
 
감사합니다 😄

**Reply by learningssik** [reply]
I know how to get it… but you can help me get badge selecting my answers??

**Reply by nel0x** [reply]
Of course it's better and more benefitial for the community if you answer real questions and there are enough out there if you hang around but if you still want to, I'd help you with that, just kindly create a discussions section on one of your Repositories and share it with us.

**Reply by learningssik** [reply]
Thanks, but i has already been fixed

**Reply by melvincwng** [reply]
@nel0x, very good point! +1 👍🏻

**Comment by nel0x** [comment]
Thank you @learningssik  very much! You really helped in this ![thread](https://github.com/Schweinepriester/github-profile-achievements/discussions/45).

As for your original question:
Just have a look at this table https://github.com/Schweinepriester/github-profile-achievements#github-profile-achievements
Galaxy Brain doesn't seem to be available now but you can do two pull requests (which have to get accepted ...) or merge two pull request on own projects without code review or **sponsor one of your favorite github projects**. All that should give you one the other achivement badges...

EDIT: Just saw that only the last one is missing for you
